### PR TITLE
Update LanguageSection.tsx

### LINKED ISSUE #834: Show C++ code if Java/Python code isn't available?
From feedback on Discord:
> i feel like you should be able to choose multiple languages instead of just 1
> because if i prefer to see sols in java, but a solution is not available in java, i have to go and change my language to c++, then afterwards change in back

This might be a problem if people don't expect the code not to be in Java/Python though (e.g. if they want to copy and paste), so how do you guys think we should approach this?

### PR DESCRIPTION
**Pull request by bqi343**
Closes #834 

Displays another lang in addition to the incomplete message.

### CODE REVIEW

File: src/components/markdown/LanguageSection.tsx [hunk 1]
```diff
@@ -47,26 +57,41 @@ export const LanguageSection = props => {
...
           </div>
         </div>
       </div>
     );
+    return (
+      <>
+        {notAvailable}
+        {fallbackLang && (
+          <>
+            <br />
```
Comment (thecodingwizard) [inline src/components/markdown/LanguageSection.tsx:87]: we probably don't need the `<br>`; otherwise the spacing looks kind of weird

**Review by thecodingwizard — APPROVED** [review]

**Comment by bqi343** [comment]
without the `<br />` there's no space between the incomplete message and the code in the solution spoiler here: https://usaco.guide/silver/binary-search?lang=cpp#example---maximum-median

**Comment by thecodingwizard** [comment]
is this better?

**Comment by bqi343** [comment]
Looks good.

**Pull request merged**